iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/bicep/TargetScopeDeclarationImplTest.java

```diff
@@ -47,12 +47,12 @@ void shouldParseSimpleTargetScopeDeclaration() {
 
   @ParameterizedTest
   @CsvSource(value = {
-    "managementGroup, MANAGEMENT_GROUP",
-    "resourceGroup,   RESOURCE_GROUP",
-    "subscription,    SUBSCRIPTION",
-    "tenant,          TENANT",
-    "other,           UNKNOWN"
-  })
+    "'managementGroup', MANAGEMENT_GROUP",
+    "'resourceGroup',   RESOURCE_GROUP",
+    "'subscription',    SUBSCRIPTION",
+    "'tenant',          TENANT",
+    "'other',           UNKNOWN"
+  }, quoteCharacter = '\"')
```

Comment:
Is it really needed?

Reply:
This was needed for Junit not to remove quotes around strings. I'm reverting this change for now